src/easier68k/core/util/parsing.py

```diff
@@ -1,5 +1,32 @@
 # Parsing utils
 
+def from_str_util(command: str, parameters: str) -> (str, list):
```

Comment:
Doesn't make much of a difference, but you could make it so that the size returned is a chr instead of a str